bluebrain/repo-bluebrain/packages/py-archngv/package.py

```diff
@@ -13,17 +13,15 @@ class PyArchngv(PythonPackage):
     git = "ssh://git@bbpgitlab.epfl.ch/nse/ArchNGV.git"
 
     version("develop", branch="main")
-    version("2.0.2", tag="ArchNGV-v2.0.2")
-    version("2.0.1", tag="ArchNGV-v2.0.1")
-    version("3.0.0", tag="ArchNGV-v3.0.0")
+    version("3.0.1", tag="ArchNGV-v3.0.1")
 
     depends_on("py-setuptools@42:", type="build")
 
     depends_on("py-numpy@1.19.5:", type=("build", "run"))
     depends_on("py-scipy@1.5.0:", type=("build", "run"))
     depends_on("py-h5py@3.1.0:", type=("build", "run"))
     depends_on("py-libsonata@0.1.8:", type=("build", "run"))
-    depends_on("py-bluepysnap@0.13:0.99", type=("build", "run"))
+    depends_on("py-bluepysnap", type=("build", "run"))
```

Comment:
According to https://bbpgitlab.epfl.ch/nse/ArchNGV/-/blob/ArchNGV-v3.0.1/setup.py?ref_type=tags
it should be:
```python
    depends_on("py-bluepysnap@1.0:", type=("build", "run"))
```

Reply:
fixed